indico/modules/events/registration/templates/display/regform_display.html

```diff
@@ -133,6 +133,7 @@
                 data-initial-values="{{ initial_values | tojson | forceescape }}"
                 data-management="{{ management | tojson | forceescape }}"
                 data-moderated="{{ moderated | tojson | forceescape }}"
+                data-lock-email="{{ invitation.lock_email | tojson | forceescape }}"
```

Comment:
oops, this broke all registration forms w/o an active invitation. luckily it showed up on the i18n-demo instance first before I put this in production (or in a release) ;)

fixed in 6efb53a7a1